### PR TITLE
Revamp the FAQ...

### PR DESCRIPTION
**Pull request by jits**
- Added a new section on how to find links to docs and resources, and merged the support request section into this, making it more prominent towards the top.
- Added in new information on projects, services and self-serve of tokens.
- Reworded a few entries.
- Made each section linkable and added links to each.
- Moved the FAQ link on the homepage before the platform themes as it's likely to be more useful to first time users.
- BONUS: Improved layout + added title for platform themes on the homepage.

### CODE REVIEW

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 1]
```diff
@@ -53,82 +87,105 @@
...
+            </ul>
           </li>
           <li>
             You can <a ui-sref="help.support.requests.overview">make a support request</a>
           </li>
         </ul>
 
         <p ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
-          If you are made a project admin, you can:
+          If you are made a <strong>project admin</strong> you can:
```
Comment (KashifSaadat) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: `are made..` -> `have been made..`?

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 2]
```diff
@@ -53,82 +87,105 @@
...
           <li>
-            Onboard and offboard team members on GitHub (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their GitHub identity within the hub.
+            Onboard and offboard team members on Github (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their Github identity within the hub.
+          </li>
+          <li>
+            Manage the <em>services</em> that are being developed in your project (via the Services tab on your project page).
+
+            <ul>
+              <li>
+                Project services are a way of grouping and breaking up your project into smaller sub-projects / deliverables / components / apps etc. Most resources provided by the platform are scoped to, and associated with, a particular service in your project (e.g. Kubernetes robot tokens).
```
Comment (KashifSaadat) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: In this area I think it would be helpful to give an example structure of the below and their associations:
- Project
- Service
- Namespaces

Maybe using ACP Team as an example, with a service Artifactory split across NotProd & Prod (fitting into the above).

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 3]
```diff
@@ -1,35 +1,69 @@
...
+          How do I create a user account on the hub?
+          <span flex></span>
+          <a class="md-icon-button" ui-sref="{'#':'how-to-create-user-account'}" target="_self"><md-icon ng-bind="$ctrl.linkIcon"></md-icon></a>
+        </span>
       </md-card-title-text>
     </md-card-title>
     <md-card-content>
       <p class="md-body-1">
-        Login <em>(Sidebar > Login)</em> using Keycloak and a new account will be created for you on the hub
+        Login <em>(Sidebar > Login)</em> using your single sign-on account (SSO) though Keycloak and a new account will be created for you on the hub
```
Comment (oyelekci) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: **though** through

Comment (marcinc) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: through

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 4]
```diff
@@ -53,82 +87,105 @@
...
     <md-card-content>
       <div class="md-body-1">
         <p>
-          You need to be granted specific permissions to be able to do certain things on the hub
+          You need to be granted specific permissions to be able to do certain things on the hub:
         </p>
         <ul>
           <li>
-            <strong>Project admins</strong> can manage a project's team and onboard/offboard team members
+            <strong>Project admins</strong> can manage a project's team, onboard/offboard team members, manage services of that project and manage Kubernetes user and robot tokens for that project
```
Comment (KashifSaadat) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: Maybe rephrase a little, duplication "of/for that project"

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 5]
```diff
@@ -1,35 +1,69 @@
...
+        </span>
+      </md-card-title-text>
+    </md-card-title>
+    <md-card-content>
+      <p class="md-body-1">
+        Both the <a ui-sref="home">homepage</a> and the side nav bar contain <em>platform themes</em> – these are related groups of information, and links to docs and resources, that you may find useful when navigating the platform and figuring out how to perform certain tasks.
+      </p>
+
+      <p class="md-body-1">
+        If you can't achieve what you need via these links or self-serve on the hub (see FAQs below for more on self-serve) then you can request support from the {{$ctrl.AppSettings.getPlatformName()}} team by filling in
```
Comment (oyelekci) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: ~~hub~~ Hub

File: platform-hub-web/src/app/help/faq/faq-entries.html [hunk 6]
```diff
@@ -53,82 +87,105 @@
...
-            You can view information and resources about the various platform <em>themes</em> from the sidebar or <a ui-sref="home">homepage</a>
+            You can view information and resources about the various <em>platform themes</em> from the sidebar or <a ui-sref="home">homepage</a>
           </li>
           <li ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
-            You can <a ui-sref="projects.list">view projects and team members</a>
+            You can <a ui-sref="projects.list">view projects and their team members</a>
+
+            <ul>
+              <li>
+                If you are a member of a project, you can also view all of it's services and associated resources (e.g. Kubernetes robot tokens for a particular service within that project)
```
Comment (marcinc) [inline platform-hub-web/src/app/help/faq/faq-entries.html]: it's services -> its services

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 1 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: 👍 will change

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 2 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: I think this is one those things that's very hard (and fairly complex) to explain in text, and more of a "you have to see and experience it to get it" – I wanted to give the bare minimum info here and then have users click around and see it for themselves. So i'm hesitant to add anymore at this stage – thinking we should first get some feedback from users and add more if necessary (and if we can come up with a really compelling example).

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 3 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: 👍

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 4 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: Agreed, will rephrase.

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 5 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: 👍

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 3 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: 👍

File: platform-hub-web/src/app/help/faq/faq-entries.html (hunk 6 above)
Comment (jits) [reply platform-hub-web/src/app/help/faq/faq-entries.html]: :thu
